docs/Contributing.asciidoc

```diff
@@ -357,11 +357,21 @@ and run `make update-deps`.  This will generate the `cpanfile` and
 The same applies to `os-autoinst` where `make update-deps` will generate the
 `cpanfile`, `os-autoinst.spec` and `docker/travis_test/Dockerfile`.
 
+If changing any package dependencies make sure packages and updated packages
+are available in openSUSE Factory and whatever current Leap version is in
+development. New package dependencies can be submitted. Before merging the
+according change into the main openQA repo the dependency should be published
+as part of openSUSE Tumbleweed.
+
+
 === Remarks regarding CI
 * The CI of os-autoinst uses the container made using `docker/travis_test/Dockerfile`.
 * The CI of openQA uses the container made using `docker/devel:openQA:ci/base/Dockerfile`
   and further dependencies listed in `.circleci/ci-packages.txt` (see
   <<Contributing.asciidoc#circleci-workflow,CircleCI documentation>>).
+* There is an additional check running using OBS to check builds of packages
+  against openSUSE Tumbleweed and openSUSE Leap
+
```

Comment:
The sentence misses a dot and I wouldn't add an additional blank line.

Reply:
ok, did that